docbook/reference/en/en-US/modules/RESTEasy_Embedded_Container.xml

```diff
@@ -73,7 +73,7 @@ public class UndertowTest
    @Test
    public void testApplicationPath() throws Exception
    {
-      server.deploy(MyApp.class);
+      server.deployOldStyle(MyApp.class);

```

Comment:
The use of deployOldStyle() seems a little unexplained. Why not just leave it as deploy()? Also, have you changed the semantics of deploy() in a way that would affect anything or surprise anyone?